include/SDL3/SDL_gpu.h

```diff
@@ -2147,6 +2147,8 @@ extern SDL_DECLSPEC SDL_GPUDevice * SDLCALL SDL_CreateGPUDevice(
  *   properties and validations, defaults to true.
  * - `SDL_PROP_GPU_DEVICE_CREATE_PREFERLOWPOWER_BOOLEAN`: enable to prefer
  *   energy efficiency over maximum GPU performance, defaults to false.
+ * - `SDL_PROP_GPU_DEVICE_CREATE_LOGDEBUGPROPS_BOOLEAN`: enable to automatically
```

Comment:
I'm kind of stuck on this property name. Maybe we instead have a CREATE_LOG_INFO_BOOLEAN property and always show the full set of info if that's true?

Reply:
I've renamed this to `SDL_PROP_GPU_DEVICE_CREATE_VERBOSE_BOOLEAN`.